welcome/setup-dev-environment.md

```diff
@@ -62,11 +62,12 @@ Before we work on the Instant Answer codebase, we first need a development machi
 
 > Already have a Codio Account? Perfect, move on to the next step.
 
-1. Go to https://codio.com and click "**Sign Up**", at the top right corner.
-2. Click "**Sign Up via GitHub**".
-3. If you aren't already signed into GitHub, enter your GitHub login details and then click "**Sign In**".
-4. Click "**Authorize application**" to continue.
-5. In the new screen, enter the required details and click "**Create Account**".
+1. Go to https://codio.com and click "**Free Trial**", at the top right corner.
```

Comment:
I think it makes more sense to just link to https://codio.com/signup/ here. It saves a step (for the user) and the documentation is then no longer subject to any aesthetic changes which Codio makes to their homepage.